src/components/doc.jsx

```diff
@@ -92,8 +92,8 @@ class Doc extends Component {
           {
             propTypes.map((propObj) => (
               <li key={propObj.propName}>
-                <b>{propObj.propName}</b>
-                <i>{`: ${propObj.type.name}`}</i>
+                <b>{propObj.propName}:</b>
```

Comment:
This is missing whitespace like @ryan-roemer pointed out. It will render `propName:propType`.